libs/designsystem/list/src/list.component.integration.spec.ts

```diff
@@ -129,6 +129,62 @@ describe('ListComponent', () => {
         });
       });
     });
+
+    describe('with shape="none"', () => {
+      beforeEach(() => {
+        spectator.setInput('shape', 'none');
+        spectator.detectChanges();
+      });
+      it('should apply transparent background to kirby-list-items', () => {
```

Comment:
Consider readability:
```suggestion
      });
      
      it('should apply transparent background to kirby-list-items', () => {
```